src/nupic/algorithms/Anomaly.cpp

```diff
@@ -109,10 +108,17 @@ Real32 Anomaly::compute(
   {
     score = (score >= binaryThreshold_) ? 1.0 : 0.0;
   }
+  // round result (to avoid near zero rounding errors)
+  score = roundf(score*(1/PRECISION))/PRECISION;
```

Comment:
@cogmission ...on the other hand, I'm looking for some nicer code for rounding to Nth decimal place than this..maybe.